CODEOWNERS

```diff
@@ -45,4 +45,4 @@
 
 /Workbooks/Virtual*Machine*/** @andrewki-msft @nasundar @rakshith210 @microsoft/applicationinsights-devtools
 
-/Workbooks/UsageMetrics/** @MSKaiLian @OsvaldoRosado @microsoft/applicationinsights-devtools
+/Workbooks/UsageMetrics/** @MSKaiLian @OsvaldoRosado @akkumari-ms @microsoft/applicationinsights-devtools
```

Comment:
for the future: you could create a "team" in github as well, and then you could maintain your team membership yourself, and have the team here instead of individual users

Reply:
cool. will add it